docs/guides/frontent-style.mdx

```diff
@@ -202,11 +202,21 @@ type MyInterface = {
 };
 ```
 
+## Testing
+
+### Avoiding snapshots
+
+Snapshot tests are a common practice, but one we avoid in Atlantis. The idea behind snapshots is great ("these tests will magically catch visual regressions") but they leave too much to human interaction, and overall understanding if it actually broke something, to be effective in practice.
+
+Chances are, if you're updating a component and a snapshot fails, you may not have the same context as the person that wrote it. Compared to it being a real assertion test that tells you what exactly is it testing.
+
+There's a much better way to write snapshot tests other than jest blob of generated code since it creates more problems and work for us. In our experience maintaining this system, we've never avoided a bug that snapshots caught, since the snapshot is autogenerated and it's so easy to miss changes that are actually breaking.
```

Comment:
@darryltec if you have more specifics or an example "better way to write snapshots" test we can link to I think this first sentence would be much more helpful 😄